frontend/src/views/shared/Widgets/WidgetBarChart.jsx

```diff
@@ -23,13 +23,33 @@ export const WidgetBarChart = ({
   info,
   legend,
   geojson,
+  type,
 }) => {
   const { rootWidgetProps, loaded, data, noData } = useWidget(
     { slug, geojson },
-    { analysisQuery, analysisBody },
+    { type, analysisQuery, analysisBody },
   );
-
   const { unit, bar_color } = useMemo(() => JSON.parse(legend), [legend]);
+  const isCOG = useMemo(() => type === 'cog', [type]);
+
+  const mergedBarData = useMemo(() => {
+    return isCOG
+      ? data?.rows
+          ?.map((d) => {
+            return { count: d.count, min: d.mappingValue };
+          })
+          .sort((a, b) => a.min - b.min)
+      : data?.rows;
+  }, [data, isCOG]);
+
+  const downloadData = useMemo(() => {
+    return isCOG
+      ? {
+          fields: ['name', 'count'],
+          rows: mergedBarData?.map((d) => ({ name: d.name, min: d.count })),
+        }
```

Comment:
I think there's an issue with the CSV. Everytime I downloaded it, the structure was like this one:
```csv
0,1
,5333
,1
,13
,4
,2051
,0
,0
,3
,0
,1
```

Here's the chart
![Analysis of Wheat [continuous cog]](https://github.com/ConservationInternational/resilienceatlas/assets/6073968/653ebd3d-75b4-4e64-9bf7-722fe97fe8e2)